grammars/fstar.cson

```diff
@@ -260,6 +260,18 @@ repository:
           { include: '#op_names' }
         ]
       }
+      {
+        begin: '`%'
+        beginCaptures:
+          0: name: 'punctuation.definition.opaque_reveal_spec.begin.fstar'
```

Comment:
TextMate scopes should use dashes to separate words, not underscores:

~~~suggestion
          0: name: 'punctuation.definition.opaque-reveal-spec.begin.fstar'
~~~